src/main/webapp/app/shared/user-settings/personal-access-tokens/personal-access-tokens.component.html

```diff
@@ -0,0 +1,47 @@
+<h1>
+    <!--Account Information-->
+    {{ 'artemisApp.userSettings.personalAccessTokens.title' | artemisTranslate }}
+</h1>
+
+<div class="form-group">
+    <label class="label-narrow" jhiTranslate="artemisApp.userSettings.personalAccessTokens.lifetime" for="field_lifetime_days">Lifetime (Days)</label>
+    <div class="input-group">
+        <input
+            required
+            type="number"
+            class="form-control"
+            [customMin]="1"
+            [customMax]="tokenMaxLifetimeDays"
+            name="lifetime"
+            id="field_lifetime_days"
+            [(ngModel)]="newTokenLifetimeDays"
+            (change)="convertTokenLifetimeFromDaysToMillis()"
+            #lifetime="ngModel"
+        />
+        <button [disabled]="!lifetime.valid || token" class="btn btn-primary btn-sm me-2 input-group-append" type="button" style="min-width: 100px" (click)="generateNewToken()">
```

Comment:
Please move custom style into a scss file or use the bootstrap classes.

Reply:
Do you have a suggestion for a style we could use?

Reply:
Mhm good point - Bootstrap doesn't have classes for min-width with concrete values :) so i think we should create a scss file since we don't want to define style in templates :thinking: 

I guess if you wan't it should also be possible with the grid system of bootstrap (e.g. container, row, col) - but it's up to you :+1: (and probably needs some additional refactoring)

Reply:
Actually, I copied the piece from another place in Artemis. I guess the same should be done there. So the global class is the best option in my opinion.

Reply:
I decided to use the local scss file.